### DISCUSSION TITLE
Đông Phương Trí Linh Kỳ Truyện ~ Thám Tử Bất Tuân Satori - SuicaoDex

### DISCUSSION THREAD

**Discussion by giscus** [TNTKien/suicaodex#17] [Manga]
# Đông Phương Trí Linh Kỳ Truyện ~ Thám Tử Bất Tuân Satori - SuicaoDex

Manga chính thức thứ 5 của Touhou, kịch bản được viết bởi ZUN và được minh họa bởi Ginmonkusei (Sau này được đổi thành Akimaki Yuu vì sức khỏe của Ginmonkusei có vấn đề) 

Bộ truyện tập trung vào Satori Komeji, một yêu quái có khả năng đọc tâm trí người khác, cô bắt đầu trở thành thám tử và đi giải quyết các vụ án ở khắp Ảo Tưởng Hương.

https://suicaodex.com/manga/f4fa3679-6918-4684-bcb6-377c9f336898

**Comment by 0xTOHO** [comment]
Đầu :)

**Reply by 0xTOHO** [reply]
Hihi mình là người đầu tên bình luận

**Comment by 0xTOHO** [comment]
<h1>hêlô</>h1
